### ISSUE TITLE
Is it possible to run as root?

### ISSUE THREAD

**Issue by leograba** [c9/core#143]
I'm trying to run /var/www as my workspace, but since most of my files there are owned by root, then a should run cloud9 as sudo to edit the files. When I try to run as root it throws an error. Is it possible to run cloud9 as root?

The output when I try to run as root (running as user is ok, I can access the directory from cloud9):

Starting standalone
Warning: running Cloud9 without using HTTP authentication.
Run using --listen localhost instead to only expose Cloud9 to localhost,
or use -a username:password to setup HTTP authentication

Connect server listening at http://192.168.1.155:8588
Using basic authentication
CDN: version standalone initialized /home/debian/c9sdk/build
Started '/home/debian/c9sdk/configs/standalone' with config 'standalone'!
VFS connection does not exist
/root/.c9/node/bin/node: 1: /root/.c9/node/bin/node: Syntax error: "(" unexpected
Error: write EPIPE
    at errnoException (net.js:905:11)
    at Object.afterWrite (net.js:721:19)
/root/.c9/node/bin/node: 1: /root/.c9/node/bin/node: Syntax error: "(" unexpected
Error: write EPIPE
    at errnoException (net.js:905:11)
    at Object.afterWrite (net.js:721:19)
/root/.c9/node/bin/node: 1: /root/.c9/node/bin/node: Syntax error: "(" unexpected
Error: EDISCONNECT: Agent disconnected
    at Parent.onDisconnect (/home/debian/c9sdk/node_modules/smith/smith.js:333:24)
    at Parent.emit (events.js:117:20)
    at Parent.Agent.disconnect (/home/debian/c9sdk/node_modules/smith/smith.js:437:10)
    at Parent.disconnect (/home/debian/c9sdk/node_modules/vfs-child/parent.js:57:39)
    at Transport.emit (events.js:92:17)
    at Socket.onDisconnect (/home/debian/c9sdk/node_modules/smith/smith.js:126:14)
    at Socket.emit (events.js:117:20)
    at _stream_readable.js:944:16
    at process._tickCallback (node.js:448:13)
/root/.c9/node/bin/node: 1: /root/.c9/node/bin/node: Syntax error: "(" unexpected
Error: read ECONNRESET
    at errnoException (net.js:905:11)
    at Pipe.onread (net.js:559:19)
/root/.c9/node/bin/node: 1: /root/.c9/node/bin/node: Syntax error: "(" unexpected

**Comment by nightwing** [comment]
Looks like /root/.c9/node/bin/node is broken. You probably need to run the installer as root too.

**Comment by leograba** [comment]
Deleted the folder /root/.c9 and tried to install as root:
sudo wget -O - https://raw.githubusercontent.com/c9/install/master/install.sh | bash
It throws an error, even though it was ok when did it previously without sudo:

--2014-07-07 00:41:38--  https://raw.githubusercontent.com/c9/install/master/install.sh
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 199.27.76.133
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|199.27.76.133|:443... connected.
ERROR: The certificate of `raw.githubusercontent.com' is not trusted.
The certificate has not yet been activated

Found this question: http://stackoverflow.com/questions/27144426/error-the-certificate-of-raw-githubusercontent-com-is-not-trusted
Even after doing sudo apt-get install ca-certificates, the error persists.

**Comment by nightwing** [comment]
i think that won't run install.sh with sudo, only wget.
Try 
```
wget https://raw.githubusercontent.com/c9/install/master/install.sh
sudo bash install.sh
```

**Comment by leograba** [comment]
The certification error persisted. Also, I tried to reinstall without sudo:

wget -O - https://raw.githubusercontent.com/c9/install/master/install.sh | bash

It seems now the same certification error is being thrown even when trying as user.

**Comment by leograba** [comment]
There was an error when installing pty.js, so I added --unsafe-perm to the install.sh line 394:
`"$NPM" install --unsafe-perm pty.js@0.2.8`

Then there was an error when installing sqlite3, so I did the same to the line 377:
`"$NPM" install --unsafe-perm sqlite3@3.0.5`

After that it stuck while installing sqlite3. I checked into .c9/node_modules and there were sqlite3 and sequelize folders. Since sequelize is installed by install.sh after sqlite3, I assumed that even with the installation stuck, it could be ok, so I tried to run server.js as sudo.

When opened the browser, the terminal where server.js was running stated that "VFS connection does not exist", but but when tried to use the IDE, everything seemed ok. 

Nightwing, thanks for your help, it was very useful!
